.pre-commit-config.yaml

```diff
@@ -31,7 +31,7 @@ repos:
       exclude: ^(docs|graphics|utils)/
 
 - repo: https://github.com/PyCQA/pylint
-  rev: v2.17.0
+  rev: v3.1.0
```

Comment:
This is an unrelated change for the PR. You will have to start another PR to make this change.